packages/grid/_modules_/grid/components/cell/GridBooleanCell.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { SvgIconProps } from '@material-ui/core/SvgIcon';
+import CloseIcon from '@material-ui/icons/Close';
+import CheckIcon from '@material-ui/icons/Check';
```

Comment:
We can't import from this package. It's not listed as a dependency because too heavy. Instead, see `components/icons/index`